docs/src/01_introduction_and_goals.adoc

```diff
@@ -2,92 +2,46 @@ ifndef::imagesdir[:imagesdir: ../images]
+* In each round, the system will generate a question.
+* In each round, the system will generate four answers, with only one being correct.
+* The system will offer registered users access to the number of games they have played.
+* The system will offer registered users access the number of questions they have answered correctly.
+* The system will offer registered users access the number of questions they have answered incorrectly.
+* The system will offer registered users access the time they have spent within the system.
+* The system will offer registered users access the ranking of the game.
+* The system will set a time limit for registered users to respond to each question.
+
+See the complete functional requirements in the xref:#section-annex[Annex] of the documentation.
 
 
-.Further Information
-
-See https://docs.arc42.org/section-1/[Introduction and Goals] in the arc42 documentation.
-
-****
-
 === Quality Goals
-
-[role="arc42help"]
-****
-.Contents
-The top three (max five) quality goals for the architecture whose fulfillment is of highest importance to the major stakeholders. 
-We really mean quality goals for the architecture. Don't confuse them with project goals.
-They are not necessarily identical.
-
-Consider this overview of potential topics (based upon the ISO 25010 standard):
-
-image::01_2_iso-25010-topics-EN.drawio.png["Categories of Quality Requirements"]
-
-.Motivation
-You should know the quality goals of your most important stakeholders, since they will influence fundamental architectural decisions. 
-Make sure to be very concrete about these qualities, avoid buzzwords.
-If you as an architect do not know how the quality of your work will be judged...
-
-.Form
-A table with quality goals and concrete scenarios, ordered by priorities
-****
+[options="header",cols="1,3"]
+|===
+|Goal|Description
+| Functional suitability | The system must fulfill its intended purpose effectively and efficiently, allowing users to register, log in, play the quiz, and access their historical data.
+| Security | The system must prioritize user data security. It must implement robust authentication mechanisms for user registration and login. The API access points for user information and generated questions must be secured with proper authorization. 
+| Reliability | The system should be reliable in generating questions from Wikidata, ensuring that questions are accurate and diverse. The system must handle user registrations, logins, and game data storage without errors.
+| Availability | The system must be available 99.99% of the time a user tries to access it.
+| Maintainability | The system must be designed and implemented in a way that facilitates easy maintenance and updates.
+| Performance efficiency | The system must deliver optimal performance, ensuring responsive interactions for users. The automatic generation of questions from Wikidata and the real-time gameplay must be efficient. The system must handle 20 concurrent users.
```

Comment:
The choice of 20 is deliberate or just random for now until we concrete it later? If it is random please write it as a variable or specify that it is a temporal datum.

Reply:
I agree with this. Also, even if it is deliberate, it seems too low of a number in my opinion: last year's en_2a team load testing was done with 3000 users.